### PR TITLE
GLTFLoader: Support multi-type nodes

### LINKED ISSUE #14875: GLTFLoader: Node does not support multiple references
Per https://github.com/KhronosGroup/glTF/issues/1433, a single node can reference up to 1 mesh, 1 camera, *AND* 1 light, not *OR*. Our current implementation picks one and ignores the others. So if a node references a camera and a light, we should just turn it into a `THREE.Group`. Also note from that thread that a node referencing a mesh could also be a joint, in which case we need to output `Bone > Mesh`.

Example: https://github.com/KhronosGroup/glTF/files/2317078/mesh-camera.zip

### PR DESCRIPTION
**Pull request by vyv03354**
Fixes #14875

### CODE REVIEW

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -2965,25 +2962,58 @@ THREE.GLTFLoader = ( function () {
...
+			var node;
+
+			// .isBone isn't in glTF spec. See .markDefs
+			if ( nodeDef.isBone === true ) {
+
+				node = new THREE.Bone();
+
+			} else if ( objects.length > 1 ) {
+
+				node = new THREE.Group();
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Nice, I like that you're using Group here!

File: examples/js/loaders/GLTFLoader.js [hunk 2]
```diff
@@ -2965,25 +2962,58 @@ THREE.GLTFLoader = ( function () {
...
 
 			} else {
 
-				return Promise.resolve( new THREE.Object3D() );
+				node = new THREE.Object3D();
 
 			}
 
-		}() ).then( function ( node ) {
+			if (node !== objects[0]) {
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Minor: could you run the linter on this file? There are a few spots that should have more whitespace, `( node !== objects[ 0 ] )`.

**Review by donmccurdy — COMMENTED** [review]
The code changes here look good, aside from the minor styleguide issue. Thanks for making this PR!

Unfortunately I'm traveling until the 15th, and won't have the chance to test this out before then. If someone else can check on the examples, then please feel free to merge it. :)

**Review by donmccurdy — APPROVED** [review]
Looks good!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**